lib/exchange.dart

```diff
@@ -480,6 +534,47 @@ class _ExchangeWidgetState extends State<ExchangeWidget> {
             context, 'error', 'failed to create order (${BeError.msg(err)})'));
   }
 
+  Future<void> _setMin() async {
+    _checkMarket(_fromAsset, _toAsset);
+    _clearSlider();
+    setState(() => _sliderSelected = AmountSliderSelected.min);
+    var value = assetAmountToUser(_fromAsset, _market.minTrade);
```

Comment:
min trade refers to the *base asset* of a market pair (eg `BTC(base asset)-NZD(quote asset)`)

so in this case if we are using the min for NZD we need to convert the `minTrade * exchangeRate` to get the NZD minTrade